demos/kfp/docling/pdf-conversion/docling_convert_pipeline.py

```
@@ -113,17 +120,19 @@ def docling_convert_and_ingest(
 
     from docling.datamodel.base_models import InputFormat, ConversionStatus
     from docling.datamodel.pipeline_options import PdfPipelineOptions
+    from docling.datamodel.pipeline_options import PdfPipelineOptions, RapidOcrOptions
```

Ruff (0.12.2)

Redefinition of unused `PdfPipelineOptions` from line 122

Remove definition: `PdfPipelineOptions`

(F811)